test/unit/testPaymentBreakdown.js

```diff
@@ -54,20 +69,44 @@ describe('PaymentBreakdown', () => {
 
         it('sets paymentPending to true if ctx.total > 0', (done) => {
             const PaymentBreakdown = steps.PaymentBreakdown;
+            const hostname = 'localhost';
             let ctx = {total: 215};
             let errors = [];
             const formdata = {};
             const session = {};
+
             /*eslint no-empty-function: 0*/
             session.save = () => {};
             co(function* () {
-                [ctx, errors] = yield PaymentBreakdown.handlePost(ctx, errors, formdata, session);
+                [ctx, errors] = yield PaymentBreakdown.handlePost(ctx, errors, formdata, session, hostname);
                 assert.deepEqual(formdata.paymentPending, 'true');
                 done();
             })
-                .catch((err) => {
-                    done(err);
-                });
+            .catch((err) => {
+                done(err);
+            });
+        });
+
+        it('sets paymentPending and createPayment to undefined if authorise fails before createPayment', (done) => {
```

Comment:
In this scenario `paymentPending` and `createPayment` are being set to `null` not `undefined`. Can we update the test description to reflect that?